src/routers/UpdateMatchAvailabilitiesRouter.ts

```diff
@@ -0,0 +1,80 @@
+import { Request } from 'express';
+import Constants from '../common/constants';
+import Availability from '../entities/Availability';
+import AvailabilityRepo from '../repos/AvailabilityRepo';
+import MatchRepo from '../repos/MatchRepo';
+import AuthenticatedApplicationRouter from '../utils/AuthenticatedApplicationRouter';
+
+class UpdateMatchAvailabilitiesRouter extends AuthenticatedApplicationRouter<void> {
+  constructor() {
+    super('POST');
+  }
+
+  getPath(): string {
+    return '/availabilities/';
+  }
+
+  async content(req: Request): Promise<void> {
+    const { matchID, schedule } = req.body;
+
+    if (!matchID) throw Error('Missing required matchID');
+    if (!schedule) throw Error('Missing required schedule list');
+
+    // Logic to update match status
+    const match = await MatchRepo.getMatchByID(matchID);
+    let status = Constants.MATCH_CREATED;
```

Comment:
can we just do `let status;` since your following if statements seem to be exhaustive?